zebra-rpc/src/methods.rs

```diff
@@ -169,6 +214,39 @@ where
         }
         .boxed()
     }
+
+    fn get_block(&self, height: String, _verbosity: u8) -> BoxFuture<Result<GetBlock>> {
+        let mut state = self.state.clone();
+        let height = Height(
+            height
+                .parse::<u32>()
+                .expect("Height as string should parse"),
```

Comment:
Panic: this `expect` will make `zebrad` panic if `lightwalletd` sends a bad height string.

Correctness: this is unchecked manual `u32` parsing. Please use the checked `Height::parse` method instead.

Can we return an error instead?
Can you also add a test with a bad height?